gym_sts/rl/mvp.py

```diff
@@ -118,6 +133,8 @@ def main(_):
         # "no_done_at_end": True,
         "model": {
             "custom_model": "masked",
+            "fcnet_hiddens": [256, 256],
```

Comment:
Isn't this just the default? Did you try other sizes?

Reply:
We tried larger sizes - this is here as a reminder